support.mdx

```diff
@@ -0,0 +1,128 @@
+
+      <Frame>
+        ![](/images/convert-oas-3.png)
+      </Frame>
+    </Accordion>
+    <Accordion title="One of my OpenAPI pages is completely blank">
+      This is usually caused by a misspelled `openapi` field in the page metadata. Make sure
+      the HTTP method and path match the HTTP method and path in the OpenAPI document exactly.
+
+      Here's an example of how things might go wrong:
+
+      ```md get-user.mdx
+      ---
+      openapi: "GET /users/{id}/"
+      ---
+      ```
+
+      ```yaml openapi.yaml
+      paths:
+        "/users/{id}":
+          get: ...
+      ```
+      Notice that the path in the `openapi` field has a trailing slash, whereas the path in the OpenAPI
+      document does not.
+
+      Another common issue is a misspelled filename. If you are specifying a particular OpenAPI document
+      in the `openapi` field, ensure the filename is correct. For example, if you have two OpenAPI
+      documents `openapi/v1.json` and `openapi/v2.json`, your metadata might look like this:
+
+      ```md api-reference/v1/users/get-user.mdx
+      ---
+      openapi: "v1 GET /users/{id}"
+      ---
+      ```
+    </Accordion>
+    <Accordion title="Requests from the API Playground don't work">
+      If you have a custom domain configured, this could be an issue with your reverse proxy. By
+      default, requests made via the API Playground start with a `POST` request to the
+      `/api/request` path on the docs site. If your reverse proxy is configured to only allow `GET`
+      requests, then all of these requests will fail. To fix this, configure your reverse proxy to
+      allow `POST` requests to the `/api/request` path.
+
+      Alternatively, if your reverse proxy prevents you from accepting `POST` requests, you can configure Mintlify to send requests directly to your backend with the `api.playground.proxy` setting in the `docs.json`, as described [here](/core-concepts/settings#param-proxy). This will
+      likely require you to configure CORS on your server, as these requests will now come directly
+      from your users' browsers.
+    </Accordion>
+  </AccordionGroup>
+</Accordion>
+
+<Accordion title="Docs page won't load (error 404)" icon="ban" iconType="regular">
+  Check to see if you have `/api` in the URL. This is a reserved folder in production environments, which produces a 404 error.
+</Accordion>
+
+
+<Accordion title="Issues adding SVG to a card" icon="image">
+We support a specific type of SVG, called JSX, and as such the image will need to be converted.
+To do this you can: 
+1. Use [this](https://react-svgr.com/playground/) converter to generate JSX compatible SVG code
+2. Make sure you only copy the code inside the `<svg/>` tag
+
+![title](/images/svg-jsx.png)
+
+3. Paste this into your card
+4. You might need to decrease the height and width to make the image fit!
+</Accordion>
+
+
+<Info>
+We do our best to respond to all requests within 24 hours but delays may occur during busy times
```

Comment:
```suggestion
We do our best to respond to all requests within 24 hours but delays may occur during busy times.
```

We'd discussed maybe moving this higher, but I'd wait until we have defined SLAs to do that.